org.jreleaser.cli.yml

```diff
@@ -0,0 +1,106 @@
+app-id: org.jreleaser.cli
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk
+appstream-compose: false
+command: jreleaser 
+finish-args:
+  - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+  - --env=JAVA_HOME=/app/jre
+  - --share=network
+  - --filesystem=host
```

Comment:
why access to the whole host?

Reply:
As stated above: "Permissions: JReleaser is a tool that creates releases for any kind of project. As such it must be able to read and write files from any directory where the user has placed their sources. For this reason --filesystem=host was added to the set of permissions."

The tool can't know for sure which directories a user would like to use for hosting their code. Should it perhaps be restricted to just "home" ?